### ISSUE TITLE
Removing file starting with # removes parent folder!

### ISSUE THREAD

**Issue by dodedodo** [Codiad/Codiad#959]
When you try to delete a file starting with # codiad will remove the parent folder of that file

I found his bug after taking the following steps.
- create a new folder containing an index.php file.
- Put these contents in the index.php file: 
```
<?php
file_put_contents('#test', '');
?>
```
- Run the php file (either via webbrowser or php console)
- Delete the generated #test file
- Check if you can find index.php anywhere.

**Comment by Andr3as** [comment]
Yes, they are related. `&` is interpreted as argument separator in the query of the url and `#` as the delimiter of a page fragment. So they need to be encoded in the file path. That's currently missing.
